solidity/app/compile.js

```diff
@@ -0,0 +1,73 @@
+const solc = require('solc')
+const fs = require('fs')
+const path = require('path')
+
+let lookupPaths = [
```

Comment:
`lookupPaths` is never reassigned. How do you feel about using `const` instead of `let`?